deployment/values.yaml

```diff
@@ -41,3 +41,7 @@ provisioner:
 istio:
   enabled: true
 
+registry:
+    repository: "ftl-registry:5000/ftl-artefacts"
```

Comment:
Possibly not for now, but should we be namespacing these by realm at some point?